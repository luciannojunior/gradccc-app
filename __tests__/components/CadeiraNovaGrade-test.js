import 'react-native';
import React from 'react';
import CadeiraNovaGrande from '../../src/components/CadeiraNovaGrade';

import renderer from 'react-test-renderer';

it('CadeiraNovaGrande selected renders correctly', () => {
  const tree = renderer.create(<CadeiraNovaGrande cadeira={selecionada=true}/>).toJSON();
  expect(tree).toMatchSnapshot();
});
it('CadeiraNovaGrande not selected renders correctly', () => {
    const tree = renderer.create(<CadeiraNovaGrande cadeira={selecionada=false}/>).toJSON();
    expect(tree).toMatchSnapshot();
});