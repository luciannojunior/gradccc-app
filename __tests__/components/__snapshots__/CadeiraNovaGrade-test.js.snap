// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CadeiraNovaGrande not selected renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "transparent",
      "borderColor": "#d6d7da",
      "borderRadius": 4,
      "borderWidth": 2,
      "flexDirection": "row",
      "height": 70,
      "margin": 10,
      "padding": 20,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    disabled={false}
    ellipsizeMode="tail"
    style={
      Object {
        "fontSize": 20,
      }
    }
  />
</View>
`;

exports[`CadeiraNovaGrande selected renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "transparent",
      "borderColor": "#d6d7da",
      "borderRadius": 4,
      "borderWidth": 2,
      "flexDirection": "row",
      "height": 70,
      "margin": 10,
      "padding": 20,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    disabled={false}
    ellipsizeMode="tail"
    style={
      Object {
        "fontSize": 20,
      }
    }
  />
</View>
`;
